src/Formats/FormatFactory.cpp

```diff
@@ -302,6 +302,26 @@ OutputFormatPtr FormatFactory::getOutputFormat(
     return format;
 }
 
+String FormatFactory::getContentType(
+    const String & name,
+    ContextPtr context,
+    const std::optional<FormatSettings> & _format_settings) const
+{
+    const auto & output_getter = getCreators(name).output_creator;
+    if (!output_getter)
+        throw Exception(ErrorCodes::FORMAT_IS_NOT_SUITABLE_FOR_OUTPUT, "Format {} is not suitable for output (with processors)", name);
+
+    auto format_settings = _format_settings ? *_format_settings : getFormatSettings(context);
+    
+    Block emptyBlock;
+    RowOutputFormatParams emptyParams;
+    WriteBufferFromOwnString emptyBuffer;
```

Comment:
by out style should be like this:
```suggestion
    RowOutputFormatParams empty_params;
    WriteBufferFromOwnString empty_buffer;
```

Reply:
Updated. I'm also working on other project on which such camel case style is adopted. I didn't switch back from it.😄